homeassistant/components/switch/tplink.py

```diff
@@ -32,31 +35,38 @@
 })
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+def setup_platform(hass: HomeAssistantType,
+                   config: Dict,
+                   add_entities: Callable[..., None],
+                   discovery_info=None) -> None:
     """Set up the TPLink switch platform."""
     from pyHS100 import SmartPlug
-    host = config.get(CONF_HOST)
-    name = config.get(CONF_NAME)
-    leds_on = config.get(CONF_LEDS)
+    host = config.get(CONF_HOST)  # Optional[str]
```

Comment:
The proper way is to use `# type: Optional[str]`, see https://www.python.org/dev/peps/pep-0526/#abstract